src/journal/ObjectRecorder.h

```diff
@@ -49,14 +50,14 @@ class ObjectRecorder : public RefCountedObject, boost::noncopyable {
     return m_oid;
   }
 
-  bool append(const AppendBuffers &append_buffers);
+  bool append(const AppendBuffers &append_buffers, bool unlock = true);
```

Comment:
Minor: I would create a "append_unlock" method that writes "append" -- the defaulted "unlock = true" hides the calling convention.